Apps/JobScheduler/src/Tasks/CopyEmailToMessagingVerification.cs

```diff
@@ -55,11 +55,8 @@ public void Run()
         {
             this.logger.LogInformation("Performing Task {Name} started.", this.GetType().Name);
 
-            int offset = 0;
-
             IQueryable<Email> query = this.dbContext.Email.Where(
-                    email => this.dbContext.MessagingVerification.Any(msgVerification => msgVerification.EmailId == email.Id && msgVerification.EmailAddress == null))
-                .OrderBy(e => e.Id);
+                email => this.dbContext.MessagingVerification.Any(msgVerification => msgVerification.EmailId == email.Id && msgVerification.EmailAddress == null));
```

Comment:
I think SonarCloud might complain if we use `.Take()` without ordering the query.

Reply:
![Screen Shot 2023-06-20 at 8 57 26 AM](https://github.com/bcgov/healthgateway/assets/58790456/f2c022bd-c1f6-4dcb-a215-fefc2be185ca)

Reply:
With the Order By, it will take 35 seconds to return 10,000 rows.

<img width="1606" alt="Screenshot 2023-06-20 at 9 52 43 AM" src="https://github.com/bcgov/healthgateway/assets/58790456/9aa53d44-9109-4b6a-ae66-3ffdd8d19c09">

Reply:
Without Order By.

<img width="1578" alt="Screenshot 2023-06-20 at 9 54 55 AM" src="https://github.com/bcgov/healthgateway/assets/58790456/a4aa861c-c80f-413b-a706-4886936edd87">